cwl/src/main/scala/cwl/CwlExpressionCommandPart.scala

```diff
@@ -10,9 +11,15 @@ import wom.values._
 case class CwlExpressionCommandPart(expr: Expression) extends CommandPart {
   override def instantiate(inputsMap: Map[LocalName, WomValue],
                            functions: IoFunctionSet,
-                           valueMapper: (WomValue) => WomValue): String = {
+                           valueMapper: (WomValue) => WomValue,
+                           runtimeEnvironment: RuntimeEnvironment ): String = {
 
-    val pc = ParameterContext.Empty.withInputs(inputsMap.map({ case (LocalName(localName), value) => localName -> value }), functions)
+    val stringKeyMap = inputsMap.map{ case (LocalName(localName), value) => localName -> value }
+
+    val pc =
+      ParameterContext(
+        runtime = runtimeEnvironment.cwlMap).
+        withInputs(stringKeyMap, functions)
```

Comment:
ToL: this formatting is a bit strange it makes it look like `withInputs` is an attribute of `ParameterContext`.
Can we do
```scala
val pc =
  ParameterContext(
     runtime = runtimeEnvironment.cwlMap
  ).withInputs(stringKeyMap, functions)
```

Reply:
Ew.  yep I'll fix this up